api/src/pcapi/routes/backoffice/utils.py

```diff
@@ -291,6 +316,36 @@ def _manage_joins(
     return query, join_log
 
 
+def get_advanced_search_args(  # type: ignore [no-untyped-def]
+    form_search_data: dict,
+    search_attributes,  # type[IndividualOffersSearchAttributes] | type[CollectiveOffersSearchAttributes]
```

Comment:
Évitons le type: ignore sur toute la fonction : 
```suggestion
def get_advanced_search_args(
    form_search_data: dict,
    search_attributes: type,  # type[IndividualOffersSearchAttributes] | type[CollectiveOffersSearchAttributes]
```